src/components/users/UserDetails.vue

```diff
@@ -35,19 +39,27 @@ import { mapActions, mapGetters } from 'vuex';
 
 export default {
   props: {
-    id: { type: [String], required: true },
+    user: { type: Object },
   },
   data() {
     return {
-      user: { account: {} },
+      userInfo: { account: {} },
     };
   },
   computed: {
     ...mapGetters({
       users: 'user/getUsers',
+      accounts: 'account/getAccounts',
     }),
   },
   mounted() {
+    this.$store.dispatch('application/setHeaderInfo', {
+      title: 'Users',
+      subtitle: 'Details',
```

Comment:
Subtitle should not show another info?

Reply:
It will be fixed in the header refactor.